src/emcpy/plots/create_map.py

```diff
@@ -0,0 +1,297 @@
+        self.domain = domain
+        self.projection = projection
+        ax = fig.add_subplot(1, 1, 1, projection=projection.projection)
+
+        if projection.__str__() not in ['npstere', 'spstere']:
+            ax.set_extent(domain.extent)
+            ax.set_xticks(domain.xticks, crs=ccrs.PlateCarree())
+            ax.set_yticks(domain.yticks, crs=ccrs.PlateCarree())
+            lon_formatter = LongitudeFormatter(zero_direction_label=False)
+            lat_formatter = LatitudeFormatter()
+            ax.xaxis.set_major_formatter(lon_formatter)
+            ax.yaxis.set_major_formatter(lat_formatter)
+
+        self.ax = ax
+
+    def add_features(self, feature_list=['coastlines']):
+        """
+        Add features onto map.
+
+        Args:
+            feature_list : (array-like, default=['coastlines']) List of cartopy
+                           cfeatures that can be found here:
+                           https://scitools.org.uk/cartopy/docs/v0.14/matplotlib
+                           /feature_interface.html
+
+
+        ***Need to get shapefiles/figure out how to get cfeatures from Cartopy
+        """
+
+        feature_dict = {
+            'coastlines': self._add_coastlines,
+            'borders': self._add_borders,
+            'states': self._add_states,
+            'lakes': self._add_lakes,
+            'rivers': self._add_rivers
+        }
+
+        for feat in feature_list:
+            try:
+                feature_dict[feat]()
+            except KeyError:
+                raise TypeError(f'{feat} is not a valid map feature.' +
+                                'Current map features supported are:\n' +
+                                f'{" | ".join(feature_dict.keys())}"')
+
+    def draw_data(self, plot_list=list()):
+        """
+        Add data layer onto map.
+
+        Args:
+            plot_list : (array-like) List of map plot objects from emcpy
+        """
+
+        plot_dict = {
+            'map_scatter': self._scatter,
+            'map_gridded': self._gridded,
+            'map_contour': self._contour
+        }
+
+        for plotobj in plot_list:
+            try:
+                plot_dict[plotobj.plottype](plotobj)
+            except KeyError:
+                raise TypeError(f'{plotobj} is not a valid plot type.' +
+                                'Current plot types supported are:\n' +
+                                f'{" | ".join(feature_dict.keys())}"')
+
+    def _scatter(self, plot):
+        """
+        Plots MapScatter object on map axes.
+        """
+
+        cs = self.ax.scatter(plot.longitude,
+                             plot.latitude,
+                             c=plot.data,
+                             s=plot.markersize,
+                             cmap=plot.cmap,
+                             marker=plot.marker,
+                             vmin=plot.vmin,
+                             vmax=plot.vmax,
+                             transform=self.projection.projection)
```

Comment:
I think this needs to be `self.projection.str_projection`.  
`str_projection` is the attribute of the `MapProjection` class
https://github.com/NOAA-EMC/emcpy/blob/4383229cdde100354ef9dce129c815dcea30c2b9/src/emcpy/plots/map_tools.py#L103

I recommend, you change `str_projection` to `projection`. The `str` is obtained by the `__str__` method.

Reply:
The transform option requires a cartopy object (i.e. `ccrs.PlateCarree()`) not a `str`. `self.projection.projection` is the cartopy object.

Reply:
Ok. No wonder I was confused. The first `projection` is an attribute of `CreateMap` object, while the 2nd `projection` is the instance of `MapProjection`

Reply:
Yes. That is probably poor variable naming on my part. Do you suggest a change? Maybe `self.proj_obj.projection`?